.github/workflows/beachball-check.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         permissions:
             contents: read # for actions/checkout to fetch code
             pull-requests: read # for dorny/paths-filter to read pull requests
-        if: github.actor != 'dependabot[bot]' && github.head_ref != 'release-staging' && github.head_ref != '3p-release-staging' && !contains(github.head_ref, 'post-release')
+        if: github.actor != 'dependabot[bot]' && github.head_ref != 'release-staging-*' && github.head_ref != '3p-release-staging' && !contains(github.head_ref, 'post-release')
```

Comment:
I'm not sure the wildcard will work in a != expression, this might need to change to a startsWith or contains